Assets/GothicVR/Scripts/Creator/MusicCreator.cs

```diff
@@ -50,6 +57,27 @@ void Start()
             musicSource = backgroundMusic.AddComponent<AudioSource>();
         }
 
+        /// <summary>
+        /// Called once every 0.20 seconds to update the music once the timer is bigger than the updateInterval
+        /// </summary>
+        void FixedUpdate()
```

Comment:
You could also think of using a Coroutine with fixed intervall size.
But to be honest: I don't see why this would break anything inside Unity. ;-)